internal/git/git.go

```diff
@@ -110,29 +110,14 @@ func Log(sha1, sha2 string) (string, error) {
 	return string(outputs), nil
 }
 
-// CurrentBranch returns the currently checked out branch and strips away all
-// but the branchname itself.
+// CurrentBranch returns the currently checked out branch
 func CurrentBranch() (string, error) {
-	cmd := New("branch")
-	cmd.Stdout = nil
```

Comment:
Error in CI is because you still need this line I'm afraid. Great looking changes tho! Love deleted code!